### PR TITLE
Enhancement/valueset/custom/organization

### PR DESCRIPTION
**Pull request by Gokulsignal**
Hi @enjoysparkling  Matt, 

- Renamed the Code system and Value Set file for License Type as part of Organization. Updated the Code system for the License Type as part of Organization.

- Created value set for signal-organization-qualification which contains expansion for 'service type' and 'license type'.

- Value set Binding for Organization.type and Organization.Qualification

Kindly review.

### CODE REVIEW

**Comment by enjoysparkling** [comment]
I don't see a reference to https://www.hl7.org/fhir/valueset-provider-taxonomy.html in the `organization.type` valueset 

See comment:  https://github.com/enjoysparkling/signal-mso-fhir-profiles/issues/60#issuecomment-1680809725

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
Can you make these changes?

File: input/vocabulary/valueset-organization-type.json
[the remark this answers is not in the document]
Comment (Gokulsignal) [reply input/vocabulary/valueset-organization-type.json:1]: The New code values are matched in the signal code system as per the changes made by Matt.

The Changes are made to the URL to reflect those from the code system.

Kindly review.

File: input/vocabulary/codesystem-organization-license-category.json
[the remark this answers is not in the document]
Comment (Gokulsignal) [reply input/vocabulary/codesystem-organization-license-category.json:1]: - The document is renamed to "signal-organization-license" and in URL and associated titles.
- hierarchyMeaning = 'is-a'  is added to the CodeSystem file. 
- concept.contains is overwritten with concept.concept .

The above changes are updated for the codesystem-organization-license.json file.

File: input/vocabulary/valueset-organization-type.json
[the remark this answers is not in the document]
Comment (enjoysparkling) [reply input/vocabulary/valueset-organization-type.json:1]: Requires change:
- Still need a `system` wherever there is a `code` element:  see vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs)

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
Requires change:
- Still need a `system` wherever there is a `code` element:  see vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs)

File: input/vocabulary/valueset-provider-taxonomy.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-provider-taxonomy.json:1]: Requires change:
- Still need a `system` wherever there is a `code` element:  see vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs)

File: input/vocabulary/valueset-provider-taxonomy.json [hunk 1]
```diff
@@ -0,0 +1,3534 @@
...
+          ]
+      }
+  ],
+  "description": "NUCC Healthcare Provider Taxonomy codes",
+  "jurisdiction": [
+      {
+          "coding": [
+              {
+                  "system": "http://unstats.un.org/unsd/methods/m49/m49.htm",
+                  "code": "001",
```
Comment (enjoysparkling) [inline input/vocabulary/valueset-provider-taxonomy.json:46]: This code should be "840" for USA

File: input/vocabulary/valueset-provider-taxonomy.json [hunk 2]
```diff
@@ -0,0 +1,3534 @@
...
+                  "display": "World"
```
Comment (enjoysparkling) [inline input/vocabulary/valueset-provider-taxonomy.json:47]: display = "United States of America"

File: input/vocabulary/valueset-signal-organization-qualification.json
[the remark this answers is not in the document]
Comment (enjoysparkling) [reply input/vocabulary/valueset-signal-organization-qualification.json:1]: Requires change:
- Still need a `system` wherever there is a `code` element:  see vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs)

And, let's leave this list as-is for now.  I will follow-up with business.

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
See notes on files.  In many places Still need a `system` wherever there is a `code` element:  see vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs)

File: input/vocabulary/valueset-organization-type.json
[the remark this answers is not in the document]
Comment (Gokulsignal) [reply input/vocabulary/valueset-organization-type.json:1]: As per the vsd-10 on ValueSet Constraints, the 'System' is already updated to the ValueSet.expansion.contains for the valueset-organization-type.json file. 

Kindly review the same: 
https://github.com/enjoysparkling/signal-mso-fhir-profiles/commit/1f1f6dc5769b9405b7657e7f3883942c7d6db86d

File: input/vocabulary/valueset-organization-license-type.json
[the remark this answers is not in the document]
Comment (Gokulsignal) [reply input/vocabulary/valueset-organization-license-type.json:1]: As per the vsd-10 on [ValueSet Constraints](https://www.hl7.org/fhir/valueset.html#invs) , the 'System' is updated to the ValueSet.expansion.contains for the valueset-organization-license-type.json file. 

Kindly review the changes.

File: input/vocabulary/valueset-provider-taxonomy.json
Comment (Gokulsignal) [reply input/vocabulary/valueset-provider-taxonomy.json:1]: For Provider taxonomy, those particular fields should be a TEXT fields as discussed. 

We don't have expansion.contain. We have only 'compose.include.concept' now. Do we need system for this Elements?

File: input/vocabulary/valueset-provider-taxonomy.json (hunk 1 above)
Comment (Gokulsignal) [reply input/vocabulary/valueset-provider-taxonomy.json:46]: The code "840" is updated for the USA in valueset-provider-taxonomy file. 

Kindly review.

File: input/vocabulary/valueset-provider-taxonomy.json (hunk 2 above)
Comment (Gokulsignal) [reply input/vocabulary/valueset-provider-taxonomy.json:47]: The 'display ' is updated as "United States of America".

Kindly review.

File: input/vocabulary/valueset-signal-organization-qualification.json
[the remark this answers is not in the document]
Comment (Gokulsignal) [reply input/vocabulary/valueset-signal-organization-qualification.json:1]: Added 'system' to expansion.contain on 'valueset-signal-organization-qualification' file as we have got an error while validating it on Forge Quality Control. 

Kindly review.

File: input/vocabulary/valueset-provider-taxonomy.json
Comment (enjoysparkling) [reply input/vocabulary/valueset-provider-taxonomy.json:1]: Why did we even specify all these codes?  Does this value set represent every code in the code system? If that is the case, we could have just defined the code system without the codes explicitly defined in the compose.include.

**Review by enjoysparkling — APPROVED** [review]
OK, I think the files are good to merge into main now.

**Pull request merged**